IN-CORE/incore-playbook
# 97 first time load view results that contains map flashes white or failed

The solution is before instantiate each widget (building damage, MCS, etc) check if all the parameters (layers, hazard etc) are ready first.

To test:
- npm run:start
- login as user `commresilience` and `Pa&&word`
- check each result popup in step 3-4
